lib/data_update_scripts/20200911045602_reindex_articles_with_videos.rb

```diff
@@ -0,0 +1,8 @@
+module DataUpdateScripts
+  class ReindexArticlesWithVideos
+    def run
+      articles = Article.where.not(video: nil).or(Article.where.not(video: ""))
```

Comment:
@rhymes Do you think this condition is sufficient?

Reply:
Wonder if the query plan is different from `Article.where.not(video: ["", nil])` but yeah! that's fine.

Our `.with_video` scope is a bit more specific but I think this should cover also those that aren't shown

https://github.com/forem/forem/blob/df245e7e84b38260be4b7fe4ef359ee539b8a40b/app/models/article.rb#L184-L189

Reply:
The generated queries are slightly different, but logically equivalent:

`WHERE ("articles"."video" IS NOT NULL OR "articles"."video" != $1)` vs `WHERE NOT (("articles"."video" = $1 OR "articles"."video" IS NULL))`

I decided against the scope because I figured we want to reindex this for all articles with videos.

Reply:
This PR should be safe for deployment, we can check on Blazer later if it has fixed the issues